src/sdk/pynni/nni/common.py

```diff
@@ -49,7 +49,11 @@ def _load_env_args():
     'debug': logging.DEBUG
 }
 
-_time_format = '%m/%d/%Y, %I:%M:%S %P'
+if sys.platform =='win32':
+    _time_format = '%m/%d/%Y, %I:%M:%S %p'
+else:
+    _time_format = '%m/%d/%Y, %I:%M:%S %P'
```

Comment:
`_time_format` the same ?

Reply:
windows use %p instead of %P to present AM or PM, I can not find %P for time.strftime. Need to fix.